src/client/stories/index.js

```diff
@@ -3,10 +3,24 @@ import { storiesOf } from "@storybook/react";
 import { withKnobs } from "@storybook/addon-knobs";
 import { MapStory } from "./Map";
 import { WithTextStory, WithEmojiStory } from "./Buttons";
+import { VesselsListStory } from "./Vessels";
+import { VesselTeaserStory } from "./VesselTeaser";
 
 storiesOf("Button", module)
   .add("with text", WithTextStory)
   .add("with emoji", WithEmojiStory);
 
 const MapStories = storiesOf("Map", module).add("MapComponent", MapStory);
 MapStories.addDecorator(withKnobs);
+
+const VesselsListStories = storiesOf("VesselList", module).add(
```

Comment:
Rename the category (storiesOf) to `Vessels` and add both `VesselTeaser` and `VesselList` under the same category.

Reply:
Renamed the category (storiesOf) to Vessels and add both VesselTeaser and VesselList under the same category.